packages/alea-frontend/pages/api/get-slides.ts

```diff
@@ -78,17 +110,25 @@ async function recursivelyExpandSlideElementsExcludeSections(
       sectionId,
     });
   }
-  return finalSlides;
+  return { slides: finalSlides, css: accumulatedCSS };
 }
 
-async function getSlidesFromToc(elems: TOCElem[], bySection: Record<string, Slide[]>) {
+async function getSlidesFromToc(elems: TOCElem[], bySection: Record<string, SlidesWithCSS>) {
   for (const elem of elems) {
     if (elem.type === 'Section') {
       const secId = elem.id;
       const slideData = await getSectionSlides(elem.uri);
       if (slideData) {
         const [css, slideElems] = slideData;
-        bySection[secId] = await recursivelyExpandSlideElementsExcludeSections(slideElems, secId);
+        const result = await recursivelyExpandSlideElementsExcludeSections(
```

Comment:
You don't need to pass the css to `recursivelyExpa...` function:
```
const [css, slideElems] = slideData;
const result = await recursivelyExpandSlideElementsExcludeSections(slideElems, secId);
const finalCss = mergeWithoutDuplicates(css, result.css);
```
The same mergeWithoutDuplicates can also be used inside `recursivelyEx...` function and make the code simpler